keps/sig-auth/2579-psp-replacement/README.md

```diff
@@ -561,16 +561,14 @@ _Note: These fields should be unconditionally restricted, regardless of targeted
 
 ### Windows Support
```

Comment:
cc @marosset @ravisantoshgudimetla, mapping this to KEP-2802 / https://github.com/kubernetes/enhancements/pull/2803

Reply:
@liggitt - What is the expectation on the new `OS` field being added to pod spec. Will the PodSecurityAdmission plugin honor it even when it is in alpha stage?

Reply:
Data will only be able to be persisted in that field if the alpha Pod OS feature is enabled. If the alpha Pod OS feature is enabled, and data is persisted in the OS field, I'd expect podsecurity admission to honor it.

Reply:
To be clear, since the PodSecurityAdmission plugin will be run by default in `privileged` mode and that profile doesn't have any OS specific fields, I presume it'd be ok.

Reply:
And for `restricted` mode we'll want to encourage users to add policy exclusions based on namespace and/or runtime class names for Windows Pods for now.

Reply:
no PodSecurity profile would *require* the OS field... if it was specified, it could be made use of

Reply:
If not, would the pod be considered Linux to be backwards compatible?

Reply:
if there's no OS specified, all PodSecurity controls should be enforced